packages/nlcst-parse-japanese/src/nlcst-parse-japanese.ts

```diff
@@ -317,9 +317,15 @@ function tokenize(
  */
 export class JapaneseParser {
     private tokenizer!: { tokenize(text: string): any };
+    private dicPath?: string;
 
-    ready() {
-        return getTokenizer().then((tokenizer: any) => {
+    constructor(options: { dicPath?: string }) {
```

Comment:
`options` would be optional.

> test/nlcst-parse-japanese-test.ts(10,25): error TS2554: Expected 1 arguments, but got 0.

The following changes will resolve this compile error.

```diff
-     constructor(options: { dicPath?: string }) {
+     constructor(options: { dicPath?: string } = {}) {
```

Reply:
Thank you for pointing that out!